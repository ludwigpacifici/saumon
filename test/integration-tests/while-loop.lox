var i = 0;
while (i < 2) {
      print i;
      i = i + 1;
}

print "Before while";
while (false) { print "This will not be printed"; }
print "End while";
