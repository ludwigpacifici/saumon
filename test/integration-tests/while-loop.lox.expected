0.
1.
Before while
End while
